lib/id3c/cli/command/etl/presence_absence.py

```diff
@@ -288,6 +291,27 @@ def update_sample(db: DatabaseSession,
     return sample
 
 
+def update_details_nwgc_id(sample: Any, additional_details: dict) -> dict:
+    """
+    Given a *sample* fetched from `warehouse.sample`,
+    extend `sample.details.nwgc_id` to an array if needed.
+
+    Add provided "nwgc_id" within *additional_details* to the existing array
+    if it doesn't already exist
+    """
+    exisiting_nwgc_ids = sample.details["nwgc_id"]
+    new_nwgc_ids = additional_details["nwgc_id"]
+
+    # Extend details.nwgc_id to an array
+    if isinstance(sample.details["nwgc_id"], int):
+        exisiting_nwgc_ids = [sample.details["nwgc_id"]]
+
+    # Concatenate exisiting and new nwgc_ids and deduplicate
+    additional_details["nwgc_id"] = list(set(exisiting_nwgc_ids + new_nwgc_ids))
+
+    return additional_details
```

Comment:
This function modifies `additional_details` in place, so returning it is unnecessary. I think it's a good habit to omit return values from functions which use in-place modification because it makes it more clear how values flow in the code.